sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Marked `VisualStudioCodeCredential` and `VisualStudioCodeCredentialBuilder` as deprecated.
```

Comment:
can this reference the github bug number in parenthesis at the end of the sentence please?